bigquery_etl/dryrun.py

```diff
@@ -69,6 +73,15 @@ def __init__(
         except FileNotFoundError:
             self.metadata = None
 
+        from bigquery_etl.cli.utils import is_authenticated
+
+        if not is_authenticated():
+            print(
+                "Authentication to GCP required. Run `gcloud auth login` "
```

Comment:
I tested and verified that the specified `gcloud auth login` command isn't sufficient without the additional `--update-adc` argument.
```suggestion
                "Authentication to GCP required. Run `gcloud auth login --update-adc` "
```
(this exact same message is used in a bunch of other places, but updating all those is probably beyond the scope of this PR).